recipes/sarge/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "sarge" %}
+{% set version = "0.1.4" %}
+{% set sha256 = "59f93216723ddd9062d17cbbb90ed9e69267b84825cf0bde0b7f8d934c424823" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - sarge
+
+about:
+  home: http://sarge.readthedocs.io/en/latest/
+  license: BSD 3-clause
+  license_family: BSD
+  summary: a wrapper for subprocess for interaction with external applications
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - takluyver
+    - fangohr
```

Comment:
@fangohr, @mb1a15, @davidcortesortuno are you ok being added as co-maintainers of the `sarge` package on conda-forge?